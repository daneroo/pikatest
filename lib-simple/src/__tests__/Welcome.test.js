import React from 'react'
import renderer from 'react-test-renderer'

import { Welcome } from '../index'

describe('Welcome', () => {
  test('import Welcome', () => {
    expect(typeof Welcome).toBe('function')
  })
  test('Welcome renders', () => {
    const component = renderer.create(
      <Welcome />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  test('Welcome renders with a param', () => {
    const component = renderer.create(
      <Welcome name={'Dude'} />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
