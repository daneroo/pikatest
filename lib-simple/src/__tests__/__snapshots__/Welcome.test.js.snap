// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Welcome renders 1`] = `
<h4>
  Hello, 
  Dan-eroo
</h4>
`;

exports[`Welcome Welcome renders with a param 1`] = `
<h4>
  Hello, 
  Dude
</h4>
`;
